lib/XML/LibXML/Dom.pm

```diff
@@ -0,0 +1,281 @@
+use v6.c;
+
+use XML::LibXML::CStructs :types;
+
+class XML::LibXML::xmlNs is xmlNs {
+
+	method setNext(xmlNs $n) {
```

Comment:
There may be hard tabs. I have recently been replacing them as I find them. 

Another old coding habit because I hated hitting right arrow through code 3 indents deep. Editors are smarter now, so it's not necessary anymore.

Old habits and all that.